crates/iota-core/src/safe_client.rs

```diff
@@ -253,7 +253,7 @@ impl<C: Clone> SafeClient<C> {
                 ))
             }
             TransactionStatus::Executed(cert_opt, effects, events) => {
-                let signed_effects = self.check_signed_effects_plain(digest, effects, None)?;
+                let signed_effects = self.check_signed_effects_plain(digest, *effects, None)?;
```

Comment:
Does this exist in the upstream change PR?

Reply:
Same here:
No, but without it, it introduced a cargo check / CI warning on our side (with our CI checks settings)
Moving the enum variant into a Box removes that warning.

Reply:
-> removed the box in the end since the problem it solves is going away in an ucoming commit port